Tests/ConduitTests/Auth/KeychainHybridKeyProviderTests.swift

```diff
@@ -0,0 +1,254 @@
+//
+//  KeychainHybridKeyProviderTests.swift
+//  Conduit
+//
+//  Created by John Hammerlund on 12/10/19.
+//
+
+import XCTest
+@testable import Conduit
+
+@available(macOS 10.12.1, iOS 10.0, tvOS 10.0, watchOS 3.0, *)
+final class KeychainHybridKeyProviderTests: XCTestCase {
+
+    private lazy var plaintextData: Data = {
+        guard let data = MockResource.json.data else {
+            fatalError("Test setup failed")
+        }
+        return data
+    }()
+
+    private var prefersSecureEnclaveStorage: Bool = {
+        // SEP access requires codesigning on Mac apps & therefore will fail unit tests. Specifically,
+        // without application entitlements, SEP operations will fail with errSecMissingEntitlements (-34018)
+        // To test this:
+        // 1. Update this property to always be 'true' (tests will be failing on macOS at this point)
+        // 2. Update ConduitTests-macOS and ConduitTestHost-macOS with Automatic development signing
+        // 3. Add Conduit.framework to the ConduitTestHost-macOS embedded frameworks (needed to codesign)
+        // 4. Add any Capability to ConduitTestHost-macOS in order to trigger generation of entitlements file
+        // 5. Clean & run tests (should pass)
+        #if os(iOS)
```

Comment:
what about tvOS and watchOS, do we want to return `true` for those?

Reply:
We could, but that property currently does nothing on either platform. If Apple ever extends the SEP to those devices, then we would want to revisit.